faq.md

```diff
@@ -138,4 +138,23 @@ In the absence of entries for headers, clangd will use
 to choose a source file whose compile command to use when opening a header.
 The heuristics are currently based on filesystem paths and can sometimes
 choose the wrong source file, though 
-[improvements](https://github.com/clangd/clangd/issues/123) are planned.
\ No newline at end of file
+[improvements](https://github.com/clangd/clangd/issues/123) are planned.
+
+## What are some other reasons clangd's diagnostics may have false positives or false negatives?
+
+To provide increased responsiveness, clangd skips parsing the bodies of
+templates defined in included headers. This optimization can result in
```

Comment:
clangd skips **function bodies**, not templates in particular.

Reply:
Thanks. I was confused because you mostly notice the template case (where a diagnostic **in the main file** is missing, because it's actually inside the body of a template defined in a header but instantiated from the main file).